app/src/main/java/com/kodeco/android/countryinfo/flow/Flows.kt

```diff
@@ -17,23 +19,36 @@ object Flows{
     private val counterFlow= MutableStateFlow(0)
     val mutableCounterFlow: StateFlow<Int> = counterFlow
 
+    private val combineFlow= MutableStateFlow(0)
+    val mutableCombineFlow: StateFlow<Int> = combineFlow
```

Comment:
note: the names here should be switched as the private val is mutable while the public val `StateFlow` is immutable:

```suggestion
    private val mutableCounterFlow= MutableStateFlow(0)
    val counterFlow: StateFlow<Int> = counterFlow

    private val mutableCombineFlow= MutableStateFlow(0)
    val combineFlow: StateFlow<Int> = combineFlow
```